src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -104,9 +104,7 @@ Copyright (c) .NET Foundation. All rights reserved.
            and VS won't show a warning icon on the reference.  See https://github.com/dotnet/sdk/issues/1499
            -->
 
-      <ReferenceCopyLocalPaths Include="@(_NETStandardLibraryNETFrameworkLib)" Condition="'%(FileName)' != 'netfx.force.conflicts'">
-        <Private>false</Private>
-      </ReferenceCopyLocalPaths>
+      <ReferenceCopyLocalPaths Include="@(_NETStandardLibraryNETFrameworkLib)" Condition="'%(FileName)' != 'netfx.force.conflicts'"/>
```

Comment:
Perhaps we should also update the comment above since it still says that Private is set to false?

Reply:
Private is still set to false on the `Reference` items which are created in the `AddFacadesToReferences` task.  That's what the comment is supposed to be referring to, as far as I understand.